pages/styles/index.css

```diff
@@ -429,7 +437,7 @@ a:-webkit-any-link {
 }
 
 .bg-blur {
-  backdrop-filter: blur(24px);
+  /* backdrop-filter: blur(24px); */
```

Comment:
@Zequez you can have this disabled on local if you want but on prod I'd like it on

Reply:
Oh I know, I must have committed it accidentally